ecs-configuration/logstash/conf.d/logstash-100-input-kafka-fsf.conf

```
@@ -5,8 +5,7 @@ input {
     # Set this to one per kafka partition to scale up
     #consumer_threads => 4
     group_id => "fsf_logstash"
-    bootstrap_servers => "127.0.0.1:9092"
-    codec => json
+    bootstrap_servers => "simplerockbuild.simplerock.lan:9092"    codec => json
```

I think this was an artifact from committing after a deploy.

```suggestion
    bootstrap_servers => "127.0.0.1:9092"
    codec => json
```